src/sesg_cli/cli/pdf_to_txt.py

```diff
@@ -0,0 +1,54 @@
+import os
+from pathlib import Path
+
+import PyPDF2
+import typer
+from rich.progress import Progress
+
+app = typer.Typer(
+    rich_markup_mode="markdown", help="Convert pdfs to txt"
+)
+
+
+@app.command()
+def convert(
+    slr_folder_path: Path = typer.Argument(
+        ...,
+        help="Path to pdfs folder to be converted",
+        dir_okay=True,
+        exists=True,
+    )
+):
+    files: list[str] = os.listdir(f"{slr_folder_path}\\pdfs")
+
+    with Progress() as progress:
+        convertion_progress = progress.add_task(
+            "[green]Converting...", total=len(files)
+        )
+        for index, file in enumerate(files):
+            paper_id: str = file.strip(".pdf")
+
+            with open(slr_folder_path / f"pdfs\\{file}", "rb") as f:
```

Comment:
if using `pathlib.Path`, this statement will change to

```python
with open(file, "rb") as f:
```

Reply:
Done!!